testsuite/mpitests/test_global_rng.sli

```diff
@@ -24,12 +24,12 @@
 /** @BeginDocumentation
 Name: testsuite::test_global_rng - check whether global rng gives consistent results
```

Comment:
`GetValue` uses the global RNG, and that is the intention of this test: to check that the global RNG gives the same values over different processes. And since it's a test, I think it's fine to keep the name. What extra documentation can be added to make it clearer?

Reply:
Could you add the fact that the test checks the global RNG by using `GetValue` as the function uses the global RNG, in the description part of the test?

Reply:
Done.